### PR TITLE
modify travis ci config 

### PR DESCRIPTION
**Pull request by zybing**
# modify travis ci config file
1. modify .travis.yml
2. add travis folder
3. modfiy /bin/run-all.sh in 29 line ,because `benchmark="${benchmark/.//}"` doesn't work well in travis ci

### CODE REVIEW

**Comment by zybing** [comment]
sorry , it's that `benchmark="${benchmark/./\/}"` doesn't work well in travis ci

File: travis/hadoop.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# The root HDFS path to store HiBench data
+hibench.hdfs.master       hdfs://localhost:9000
+
+
+# Hadoop release provider. Supported value: apache, cdh5, hdp
+hibench.hadoop.release    hdp
+
+#Hadoop version. Only hadoop2 is supported
+#TODO: remove this
+hibench.hadoop.version    hadoop2
```
Comment (carsonwang) [inline travis/hadoop.conf]: The above three lines can be removed.

**Comment by carsonwang** [comment]
I'll merge this first. Thanks @zybing . 
The hibench.conf and spark.conf has been updated in the master branch. Can you include those in the conf? Can we also test building with spark=2.0?

**Comment by zybing** [comment]
ok @carsonwang I will modify project as your advices.

**Pull request merged**